src/main/scala/com/ibm/developer/eventstore/akka/OnlineRetailWebServer.scala

```diff
@@ -19,62 +19,73 @@ import java.nio.charset.StandardCharsets
 import java.sql.Timestamp
 
 import akka.Done
-import akka.actor.{ ActorSystem, Terminated }
-import akka.event.Logging
-import akka.util.ByteString
-import akka.http.scaladsl.model.ws.{ BinaryMessage, Message, TextMessage }
-import akka.http.scaladsl.model.{ ContentTypes, HttpEntity }
-import akka.http.scaladsl.server.{ HttpApp, Route }
-import akka.http.scaladsl.settings.ServerSettings
-import akka.stream.{ ActorMaterializer, ActorMaterializerSettings, Supervision }
-import akka.stream.alpakka.csv.scaladsl.{ CsvParsing, CsvToMap }
+import akka.http.scaladsl.model.ContentTypes
+import akka.http.scaladsl.model.HttpEntity
+import akka.http.scaladsl.model.ws.BinaryMessage
+import akka.http.scaladsl.model.ws.Message
+import akka.http.scaladsl.model.ws.TextMessage
+import akka.http.scaladsl.server.HttpApp
+import akka.http.scaladsl.server.Route
+import akka.stream.Materializer
+import akka.stream.alpakka.csv.scaladsl.CsvParsing
+import akka.stream.alpakka.csv.scaladsl.CsvToMap
 import akka.stream.alpakka.ibm.eventstore.scaladsl.EventStoreSink
-import akka.stream.scaladsl.{ Flow, Keep, Source }
+import akka.stream.scaladsl.Flow
+import akka.stream.scaladsl.Keep
+import akka.stream.scaladsl.Sink
+import akka.util.ByteString
 import com.ibm.event.common.ConfigurationReader
 import org.apache.spark.sql.Row
 
-import scala.concurrent._
-import scala.util.Success
-
-class OnlineRetailWebServer extends HttpApp {
-
-  val decider: Supervision.Decider = { e =>
-    println("Unhandled exception in stream", e)
-    Supervision.Resume
-  }
-
-  implicit val system: ActorSystem = ActorSystem(Logging.simpleName(this).replaceAll("\\$", ""))
-  val materializerSettings: ActorMaterializerSettings = ActorMaterializerSettings(system).withSupervisionStrategy(decider)
-  implicit val materializer: ActorMaterializer = ActorMaterializer(materializerSettings)
-  implicit val executionContext: ExecutionContextExecutor = system.dispatcher
+import scala.concurrent.Future
 
-  ConfigurationReader.setConnectionEndpoints("0.0.0.0:1100")
+object OnlineRetailWebServer extends HttpApp {
 
-  private val shutdownPromise = Promise[Done]
-  private val db = "TESTDB"
-  private val tableName = "OnlineRetailOrderDetail"
-  private val cancelTableName = "OnlineRetailCancelDetail"
+  private val DB = "TESTDB"
+  private val TableName = "OnlineRetailOrderDetail"
+  private val CancelTableName = "OnlineRetailCancelDetail"
```

Comment:
Constants in Scala are in general starting with capital and then camel cased

Reply:
Also: these probably belong in the application.conf file rather than hardcoded, but a bit tricky when using `HttpApp ` so I left them.